tests/commands/test_build.py

```diff
@@ -462,8 +474,31 @@ def test_build_multiple_with_charmcraft_yaml_managed_mode(
     )
 
 
+@pytest.mark.parametrize(
+    "charmcraft_yaml, metadata_yaml",
+    [
+        [
+            dedent(
+                """\
+                type: charm
+                """
+            ),
+            dedent(
```

Comment:
Since `basic_project` has this metadata.yaml, do we need to include it?

Reply:
Following PR needs it.